modules/custom/az_person/az_person_profile_import/az_person_profile_import.module

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * Contains az_person_profile_import.module.
+ */
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\node\Entity\Node;
+
+/**
+ * Implements hook_entity_presave().
+ */
```

Comment:
I wonder if this could be done in an Event Subscriber, on the [onPreRowSave event](https://api.drupal.org/api/drupal/core%21modules%21migrate%21src%21Event%21MigrateEvents.php/constant/MigrateEvents%3A%3APRE_ROW_SAVE/10)

Reply:
I'm not sure if this is possible currently. On testing this out, the migration `OnPreRowSave` appears to only have access to the migration row, not the entity, for access to the revision log settings. it also seems like the Entity side of Drupal currently [lacks the equivalent entity events for subscribers to interact with](https://www.drupal.org/project/drupal/issues/2551893). That seems surprising...

It seems like an issue in progress also for entity migrations to interact with revisions in a more direct way: [https://www.drupal.org/project/drupal/issues/2988382](https://www.drupal.org/project/drupal/issues/2988382)